### PR TITLE
UIDATIMP-511 Field mappings: Reference dropdowns - long lists display…

### PR DESCRIPTION
**Pull request by IvanKryzhanovskyi**
## Overview
In Instance, Holdings, and Item field mapping profiles, whenever there are long reference list dropdowns, they don't display properly

## Approach
* Add class to reference dropdowns list
* Add styles for new class to restrict dropdowns height

## Ticket
[UIDATIMP-511](https://issues.folio.org/browse/UIDATIMP-511)

## Before
dropdown_before

## After
dropdown_after

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 * Change placement of repeatable decorator in the Inventory field mapping screens (UIDATIMP-518)
 * Implement delete row functionality for the Field Mapping Profile details (UIDATIMP-498)
 * Implement re-order row functionality for the Field Mapping Profile details (UIDATIMP-496)
+* Reference dropdowns long lists displaying correctly (UIDATIMP-511)
```
Comment (IhorKoba) [inline CHANGELOG.md:16]: Seems like it's a bug, not a story. So, could you please move description to bugfix section

Comment (IvanKryzhanovskyi) [reply CHANGELOG.md]: done

Comment (mariia-aloshyna) [inline CHANGELOG.md]:
```suggestion
* Fix reference dropdowns long lists height (UIDATIMP-511)
```

Comment (IvanKryzhanovskyi) [reply CHANGELOG.md]: Done

**Pull request merged**